app/components/ServicesSectionContent/ServicesSectionContent.tsx

```diff
@@ -1,66 +1,78 @@
-import { Chip } from "components";
+import { useContext } from "react";
+import { Chip, MobileNavigation } from "components";
 import { ServicesCard } from "./ServicesCard";
+import { BurgerContext } from "context";
 
 export const ServicesSectionContent = () => {
+  const { activeBurger } = useContext(BurgerContext);
```

Comment:
please use this [guide](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) to name boolean variables